hca/upload_to_cloud.py

```diff
@@ -89,6 +89,8 @@ def upload_to_cloud(files, staging_bucket, replica, from_cloud=False):
                     "hca-dss-content-type": _mime_type(fh.raw.name)
                 }
 
+                raw_fh.close()
```

Comment:
I would go further and say that I don't understand why this function is closing the file handles. Closing the file handles should be the caller's concern, it shouldn't incur this side effect.